labels.mss

```diff
@@ -46,8 +53,8 @@
   }
 }
 #city[capital='yes'][zoom>=5][zoom<=10],
-#city[zoom>=7][zoom<=10],
-#place_low[capital='yes'][zoom>=5][zoom<=10],
+#city[zoom>=5][zoom<=10],
+#place_low[capital='yes'][zoom>=7][zoom<=10],
```

Comment:
I think this row could actually be removed: I don't think we want to deal with capitals which are not in our own data file anyway.

Reply:
+1 but if we have capitals in place_low that are not in our file they won't appear as city, so we should add them in our file without the capital flag, isn't it?

Reply:
Oh, actually, now that we start at zoom 7 instead of 5 this row is exactly redundant with the one below ;)